### ISSUE TITLE
Random CTD

### ISSUE THREAD

**Issue by elijahlorden** [raoulvdberge/refinedstorage#1179]
#### Issue description:
I was using the Crafting Grid like normal, and got a random CTD with the attached crashlog.
#### What happens:
Crash to desktop

#### What you expected to happen:
...Not a CTD?

#### Steps to reproduce:
I have been unable to reproduce the error.

#### Version (Make sure you are on the latest version before reporting):

- Minecraft: 1.10.2
- Forge:
- Refined Storage:

Does this issue occur on a server? [yes/no]
Yes

#### If a (crash)log is relevant for this issue, link it here:
[crash-2017-04-21_12.00.40-client.txt](https://github.com/raoulvdberge/refinedstorage/files/947625/crash-2017-04-21_12.00.40-client.txt)

**Comment by elijahlorden** [comment]
Ah, I think I have figured out what is causing the issue.  We uninstalled SuperCircuitMaker from the server, but I forgot to remove it from my client.  The crash occurred after clearing the crafting grid's crafting inventory of one piece of redstone dust, which displayed the recipe for tiny redstone dust, even though that item did not exist on the server.  I think this may be an issue of information mismatch.
